libraries/llvm/io.c

```diff
@@ -142,12 +144,14 @@ void c_fs_read(Int fd, struct Pos buffer, Int offset, Stack stack) {
 
 void c_fs_write(Int fd, struct Pos buffer, Int offset, Stack stack) {
 
+    char* bytes = c_buffer_bytes(buffer); // libuv expects signed integers
+    uv_buf_t buf = uv_buf_init(bytes, c_buffer_length(buffer));
+    // erasePositive(buffer);
+    // TODO we should erase the buffer but abort if this was the last reference
```

Comment:
You mean: decrement the RC but don't free?

Reply:
Postponed solution:

a)
- allocate a user-defined struct `Stack x Buffer`
- erase in the continuation / callback

b) 
- erase here, but make sure that it is USED on the Effekt side.